examples/darwin-framework-tool/commands/provider/OTAProviderDelegate.mm

```diff
@@ -113,14 +117,18 @@ - (void)handleApplyUpdateRequest:(MTROtaSoftwareUpdateProviderClusterApplyUpdate
 }
 
 - (void)handleNotifyUpdateApplied:(MTROtaSoftwareUpdateProviderClusterNotifyUpdateAppliedParams * _Nonnull)params
+                       controller:(MTRDeviceController *)controller
+                           nodeID:(NSNumber *)nodeID
                 completionHandler:(StatusCompletion _Nonnull)completionHandler
 {
     completionHandler(nil);
 }
 
-- (void)handleBDXTransferSessionBegin:(NSString * _Nonnull)fileDesignator
-                               offset:(NSNumber * _Nonnull)offset
-                    completionHandler:(void (^)(NSError * error))completionHandler
+- (void)handleBDXTransferSessionBeginForNodeID:(NSNumber * _Nonnull)nodeID
```

Comment:
nit: All the other methods have nodeID first, then controller, maybe keep them all consistent?

Reply:
Yes, maybe I should just put the ForNodeID first on all of them...